tests/ui/requirements.txt

```diff
@@ -1,3 +1,4 @@
+django_waffle==0.11.1
```

Comment:
As we're now using `live_server`, the tests require pytest-django and therefore we should probably just import all of the dev requirements here.